enumfields/enums.py

```diff
@@ -39,6 +41,33 @@ def __new__(mcs, name, bases, attrs):
 
 class ChoiceEnum(ChoiceEnumMeta('ChoiceEnum', (BaseEnum,), _EnumDict())):
 
+    @classmethod
+    def get_allowed_previous_choices(cls, current_choice):
+        return cls._get_previous_map().get(current_choice) or []
+
+    @classmethod
+    @lru_cache(maxsize=1)
+    def get_initial_choices(cls):
+        return [
+            choice for choice in cls
+            if not hasattr(choice, 'initial') or choice.initial
```

Comment:
Why not to use just `getattr`?

Reply:
I've added property to Choice class so it;s always there.